prime-angular-frontend/src/app/modules/site-registration/pages/overview/overview.component.ts

```diff
@@ -97,4 +110,27 @@ export class OverviewComponent implements OnInit {
       this.showSubmission = !this.site.submittedDate;
     }
   }
+
+  /**
+   * @description
+   * Infer whether user was required to accept an Organization Agreement.
+   * User saw an Organization Agreement if the current Site has a care setting that is unique
+   * for that Organization.
+   */
+  private determineOrgAgreementRequired$(): Observable<SiteListViewModel[]> {
+    const currentCareSettingCode: number = this.site.careSettingCode;
```

Comment:
We do have an enum (CareSettingEnum) we can use here to have stronger typing on these variables

Reply:
Addressed in latest commit 68fb52e